File: fixtures/hub-cobol-clbs-mini/batch/reference_emit.py
"""Reference emit for CLBSMATH — same inputs as fixtures/hub-cobol-clbs-mini/batch/CLBSMATH.cbl."""
AMOUNT = 1000.00
RATE = 0.0525
RESULT = round(AMOUNT * RATE + 1e-12, 2)
print(f"{RESULT:.2f}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit.java
// EXPECTED: 52.50
// Reference emit contract for CLBSMATH (1000.00 * 0.0525 rounded).
public final class ClbsMathRef {
  public static void main(String[] args) {
    double amount = 1000.00;
    double rate = 0.0525;
    double result = Math.round(amount * rate * 100.0) / 100.0;
    System.out.printf("%.2f%n", result);
  }
}
